tfx/examples/bert/mrpc/bert_mrpc_pipeline.py

```diff
@@ -0,0 +1,200 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MRPC Classification example using TFX."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+from typing import List, Text
+
+import absl
+import tensorflow_model_analysis as tfma
+
+from tfx.components.example_gen.import_example_gen.component import ImportExampleGen
+from tfx.components import Evaluator
+from tfx.components import ExampleValidator
+from tfx.components import Pusher
+from tfx.components import ResolverNode
+from tfx.components import SchemaGen
+from tfx.components import StatisticsGen
+from tfx.components import Trainer
+from tfx.components import Transform
+from tfx.components.trainer.executor import GenericExecutor
+from tfx.components.base import executor_spec
+from tfx.dsl.experimental import latest_blessed_model_resolver
+from tfx.orchestration import metadata
+from tfx.orchestration import pipeline
+from tfx.orchestration.beam.beam_dag_runner import BeamDagRunner
+from tfx.proto import example_gen_pb2
+from tfx.proto import pusher_pb2
+from tfx.proto import trainer_pb2
+from tfx.types import Channel
+from tfx.types.standard_artifacts import Model
+from tfx.types.standard_artifacts import ModelBlessing
+from tfx.utils.dsl_utils import tfrecord_input
+
+_pipeline_name = 'bert_mrpc'
+
+# This example assumes the utility function is in ~/bert/mrpc
```

Comment:
Nit: update comment string to match [other examples](https://github.com/tensorflow/tfx/blob/master/tfx/examples/mnist/mnist_pipeline_native_keras.py#L46).